GeneratorLib/CodegenTypeFactory.cs

```diff
@@ -11,45 +11,51 @@ public static CodegenType MakeCodegenType(string name, Schema schema)
         {
             var codegenType = InternalMakeCodegenType(Helpers.ParsePropertyName(name), schema);
 
-            if (schema.Required)
+            if (schema.IsRequired)
             {
                 codegenType.Attributes.Add(new CodeAttributeDeclaration(new CodeTypeReference(typeof(JsonRequiredAttribute))));
             }
 
-            codegenType.Attributes.Add(new CodeAttributeDeclaration(new CodeTypeReference(typeof(JsonPropertyAttribute)), new []{ new CodeAttributeArgument(new CodePrimitiveExpression(name)) }));
+            codegenType.Attributes.Add(new CodeAttributeDeclaration(new CodeTypeReference(typeof(JsonPropertyAttribute)), new[] { new CodeAttributeArgument(new CodePrimitiveExpression(name)) }));
 
             return codegenType;
         }
 
         private static CodegenType InternalMakeCodegenType(string name, Schema schema)
         {
+            if (schema.Not != null)
+            {
+                // TODO implement this for glTF 2.0
```

Comment:
I left TODO breadcrumbs for features that are in glTF 2.0, but not required for loading. They center around validation.